### DISCUSSION TITLE
Welcome to arrow-flight-cli Discussions!

### DISCUSSION THREAD

**Discussion by mattphelps-8451** [8451/arrow-flight-cli#9] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by mattphelps-8451** [comment]
My name's Matt Phelps, I currently work for 84.51 as an engineer and I think @KovacicAi  has done some awesome work as the lead contributor on this CLI client for Arrow flight servers!

Also, here are three emojis to describe my hobbies! 🎹🎾🇮🇹

**Comment by aidankovacic-8451** [comment]
My name is Aidan Kovacic. I am a Computer Engineering student at the University of Cincinnati who previously worked with @mattphelps-8451 at 84.51 as a software engineer co-op. I am excited to be able to continue to work on this project alongside the open source community!

Some emojis to describe my hobbies: 🚲🥾 🇯🇵
